### PR TITLE
Add new IC blog post

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
In this PR, I add a new blog post about illumination correction and included multiple figures.

Note: I do include the convention `` to show the images in the markdown on VSCode, but this will be removed prior to merging so that it does not interfere with how it looks on the website.

### CODE REVIEW

File: _posts/2023-08-04-illumsteps.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+## Contents
+
+Within this blog post, I will be going over:
+
+1. 🪜 Basic steps to illumination correction
+2. 💡 Overview on CellProfiler IC
+3. 🎙️ An updated opinion on IC methods
+
+**Note:** For figures in the first two sections, I will be utilizing the example images provided by CellProfiler in [their tutorial for illumination correction](https://cellprofiler.org/examples).
```
Comment (MikeLippincott) [inline _posts/2023-08-04-illumsteps.md:28]: Consider adding this as sub bullt points hyperlined as `Data used in examples`

File: _posts/2023-08-04-illumsteps.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+  image="images/blog/illumsteps/fig_3.png"
+  width="100%"
+%}
+
+### Step 3: Repeat for each _channel_
+
+For all datasets you work with, these basic steps should be taken as quality control assurance prior to analysis.
+
+The biggest note is that traditionally, an IC function is created for each `channel`, not all of the images in a dataset. This is due to different distributions and sizes of the objects in a channel.
+As well, I have noticed myself that for some unknown reason to me, there are patterns of uneven illumination to seem to occur only in images in the same channel, but this is just my experience.
```
Comment (MikeLippincott) [inline _posts/2023-08-04-illumsteps.md:106]: This sentence flows a bit funky, consider rewording. I think it is the double use of myself and me?

File: _posts/2023-08-04-illumsteps.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+If these steps are not performed, you will likely have a hard time finding optimal segmentation parameters and the intensity measurements will not be biologically accurate.
+
+**NOW** that you know the steps to determining if IC is needed and/or if an IC method worked, lets get into how to create an IC function through CellProfiler!
+
+---
+
+## Creating a CellProfiler IC function
+
+You might be thinking: well, you already said that you are using images from an already existing tutorial from CellProfiler, why do you need to create _ANOTHER_ tutorial?
```
Comment (MikeLippincott) [inline _posts/2023-08-04-illumsteps.md:116]:
```suggestion
You might be thinking: "Well, you already said that you are using images from an already existing tutorial from CellProfiler, why do you need to create _ANOTHER_ tutorial?".
```

**Review by MikeLippincott — COMMENTED** [review]
GREAT post, I like the mix between information and narration :D

File: _posts/2023-08-04-illumsteps.md (hunk 2 above)
Comment (jenna-tomkinson) [reply _posts/2023-08-04-illumsteps.md:109]: Now that I am reading it, I am not a fan and don't think it makes sense. Thank you for bringing it up!

File: _posts/2023-08-04-illumsteps.md (hunk 3 above)
Comment (jenna-tomkinson) [reply _posts/2023-08-04-illumsteps.md:119]: Good catch, just implemented!

**Comment by jenna-tomkinson** [comment]
@MikeLippincott and @gwaybio 

Thank you for the reviews! I think this blog is ready to be posted. Thank you!

**Pull request merged**